src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -53,6 +59,14 @@ public static TestIndexResourcesManager getInstance() {
         return instance;
     }
 
+    public  Index getServerlessIndexConnection() {
```

Comment:
Cool, for a lot of my tests I was using the `getServerlessIndexName` / `getPodIndexName` calls to get the name and then call `.getIndexConnection()` or `.getAsyncIndexConnection` in the test file. This feels nicer though since it keeps the index'y pieces together. I can update my tests with this when it's merged. 👍

Reply:
yeah!